lib/cloudevents.ex

```diff
@@ -105,6 +105,13 @@ defmodule Cloudevents do
     {:ok, reply}
   end
 
+  # ---
+
+  defp registry_url_with_http_schema("http://" <> _registry_host = registry_url), do: registry_url
+  defp registry_url_with_http_schema("https://" <> _registry_host = registry_url), do: registry_url
+  defp registry_url_with_http_schema(registry_url) when is_binary(registry_url), do: "http://" <> registry_url
+  defp registry_url_with_http_schema(registry_url), do: registry_url
```

Comment:
the last case here is just `nil`, right? Perhaps it'd be better to make that explicit, e.g.,

```elixir
defp registry_url_with_http_schema(nil), do: nil
```

Reply:
Yes, it's `nil`. Okay, was thinking what would be better.

Reply:
I guess it's better to let it crash if it's something we didn't expect